pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -337,7 +361,7 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 
 		// Update actual state of world
 		addVolumeNodeErr := actualStateOfWorld.MarkVolumeAsAttached(
-			v1.UniqueVolumeName(""), volumeToAttach.VolumeSpec, volumeToAttach.NodeName, devicePath)
+			v1.UniqueVolumeName(""), originalSpec, volumeToAttach.NodeName, devicePath)
```

Comment:
Im wondering why do we need to `MarkVolumeAsAttached` with `originalSpec` but not migrated spec?